package.json

```diff
@@ -36,19 +37,22 @@
 	"scripts": {
 		"build": "esbuild src/content-script.ts --bundle --minify --outfile=lib/content-script.js --sourcemap",
 		"compile": "tsc",
+		"firefox:dev": "web-ext run",
+		"firefox:package": "web-ext build",
```

Comment:
[Documentation] The existing `build` task does a build. And then these also are kind of builders? It's not clear from the docs whether `pnpm run build` needs to come before `pnpm run firefox:package`.

Suggestion: how about...

* Briefly explaining this in the docs
* Calling these `dev:web-ext` (to follow `dev` naming convention from Astro & Next.js) and `build:web-ext`, respectively

Thoughts?

Reply:
Sounds good to me.